src/relay/backend/build_module.cc

```diff
@@ -38,26 +38,23 @@ namespace tvm {
 namespace relay {
 namespace backend {
 
+using TargetsMap = Map<tvm::Integer, tvm::Target>;
+
 /*!
  * \brief Context name / index
  *        See: python/tvm/_ffi/runtime_ctypes.py
  */
 struct ContextMap {
   static const std::unordered_map<int, std::string> mask2str;
-  static const std::unordered_map<std::string, int> str2mask;
   static std::string Mask2Str(int mask) {
     CHECK_GT(mask2str.count(mask), 0) << "Unknown mask.";
     return mask2str.at(mask);
   }
-  static int Str2Mask(const std::string& str) {
-    CHECK_GT(str2mask.count(str), 0) << "Unknown context.";
-    return str2mask.at(str);
-  }
 };
 
 const std::unordered_map<int, std::string> ContextMap::mask2str = {
-  {1, "cpu"},
-  {2, "gpu"},
+  {1, "llvm"},
```

Comment:
The original mapping seems to be correct, any reason for this change?

Reply:
We actually need to create `Target` from device type here: https://github.com/dmlc/tvm/blob/master/src/relay/backend/build_module.cc#L599. So I changed it from context name to target string. I can probably change the variable name to `ContextTargetMap` as well.